parley/src/layout/data.rs

```diff
@@ -214,6 +205,14 @@ impl LineItemData {
         self.bidi_level & 1 != 0
     }
 
+    pub(crate) fn compute_line_height<B: Brush>(&self, layout: &LayoutData<B>) -> f32 {
```

Comment:
```suggestion
    pub(crate) fn line_height<B: Brush>(&self, layout: &LayoutData<B>) -> f32 {
```

nit: There isn't so much computation happening here